src/ResourceManager/IotHub/AzureRM.IotHub.psd1

```diff
@@ -63,7 +63,7 @@ RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '3.2.1'; }
 # TypesToProcess = @()
 
 # Format files (.ps1xml) to be loaded when importing this module
-FormatsToProcess = '.\Microsoft.Azure.Commands.IotHub.format.ps1xml'
+#FormatsToProcess = '.\Microsoft.Azure.Commands.IotHub.format.ps1xml'
```

Comment:
@chieftn I would change this to

```psd1
# FormatsToProcess = @()
```

for consistency across all empty/commented out attributes in this file